QueryManager/src/main/java/eu/neclab/ngsildbroker/queryhandler/controller/QueryController.java

```diff
@@ -69,10 +74,14 @@ public class QueryController {
 	@Path("/entities/{entityId}")
 	@GET
 	public Uni<RestResponse<Object>> getEntity(HttpServerRequest request, @QueryParam(value = "attrs") String attrs,
-			@QueryParam(value = "options") String options, @QueryParam(value = "lang") String lang,
-			@QueryParam(value = "geometryProperty") String geometryProperty,
-			@QueryParam(value = "localOnly") boolean localOnly, @PathParam("entityId") String entityId,
-			@QueryParam(value = "doNotCompact") boolean doNotCompact) {
+											   @QueryParam(value = "options") String options, @QueryParam(value = "lang") String lang,
+											   @QueryParam(value = "geometryProperty") String geometryProperty,
+											   @QueryParam(value = "localOnly") boolean localOnly, @PathParam("entityId") String entityId,
+											   @QueryParam(value = "doNotCompact") boolean doNotCompact,
+											   @QueryParam("containedBy")@DefaultValue("") String containedBy,
```

Comment:
Why are you adding "@DefaultValue("")" here. pls validate

Reply:
because @DefaultValue  expected string as argument. validation done